src/plugins/data/public/search/session/sessions_mgmt/components/table/table.test.tsx

```diff
@@ -149,16 +149,16 @@ describe('Background Search Session Management Table', () => {
     });
   });
 
-  // FLAKY: https://github.com/elastic/kibana/issues/88928
-  describe.skip('fetching sessions data', () => {
+  describe('fetching sessions data', () => {
     test('re-fetches data', async () => {
-      jest.useFakeTimers({ legacyFakeTimers: true });
-      sessionsClient.find = jest.fn();
+      jest.useFakeTimers();
+      const find = jest.fn();
+      sessionsClient.find = find;
       mockConfig = {
         ...mockConfig,
         management: {
           ...mockConfig.management,
-          refreshInterval: moment.duration(10, 'seconds'),
+          refreshInterval: moment.duration(1, 'seconds'),
```

Comment:
I think this is the default from the top of the file, so it maybe isn't needed here?

Reply:
thx, good 👁️ , removed it